### ISSUE TITLE
SVG icon pack support.

### ISSUE THREAD

**Issue by Livid96** [Kunzisoft/KeePassDX#1880]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by Bugg4** [comment]
Just to be sure so I don't open a duplicate issue: this is also referring to single custom icons you can apply from this screen, correct?

Because currently it only lets me upload raster icons.

![Screenshot_20250908-122028.png](https://github.com/user-attachments/assets/7a54b734-4bae-43aa-9a02-939ec54e4da2)

**Comment by J-Jamet** [comment]
Currently, no SVG format is recognized for icons, but you're right, we could improve the import to be able to transform it.
The main issue is making the KeePass format compatible with SVG icons for a native vector management.

**Comment by livid96** [comment]
What I meant was to be able to use an SVG zip icon pack in this app, just like Aegis Authenticator does. 

(Easier than importing one by one manually.)

Aigis Auth: https://github.com/beemdevelopment/Aegis


Aegis Icon: https://github.com/Delta-Icons/aegis

![Screenshot_20250909-235717.png](https://github.com/user-attachments/assets/2083e399-eec9-4ad6-9b54-8ceb07e062be)

![Screenshot_20250909-235638.png](https://github.com/user-attachments/assets/6d33c73a-f0b0-472f-9c17-b5040946f28c)

**Comment by jksekai** [comment]
so will it ever be added?

**Comment by livid96** [comment]
I don’t think they will add it, It's been so long since i made this request😐

**Comment by J-Jamet** [comment]
The fact that an issue was opened a long time ago does not mean that it has been abandoned. Only that I am obliged to treat by priority.

**Comment by livid96** [comment]
Yeah, since the issue is still open, i just shouldn’t say somthing like this, Sorry.
